tools/rosbash/scripts/rosrun

```diff
@@ -16,13 +16,13 @@ function debug() {
 
 while [ $args == 1 ]
 do
-  case $1 in
+  case "$1" in
```

Comment:
Actually, at this point the arguments may also be options (`--help` or `--prefix`). Not that those contain spaces, but the user could still type them. Even on invalid input the program should behave nice and display the proper error message, I think.

Though as it turns out bash doesn't do word splitting on the argument to `case` so it isn't necessary here. Still, it can't hurt either.